src/App.tsx

```diff
@@ -1,15 +1,37 @@
-import React from 'react';
+import { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState<string>('');
+
+  const filteredFilms = () => moviesFromServer.filter(movie => {
+    const { title, description } = movie;
+    const serchedTitle = title.toLowerCase();
+    const inputQuery = query?.trim().toLowerCase();
+
+    return (
+      serchedTitle.includes(inputQuery)
```

Comment:
move these checks to the separate function
i mean
`string` includes `substring` & reuse it
Also, you can move normalizing variables to this function

Reply:
Done, I guess, I think, maybe...